dev-packages/browser-integration-tests/loader-suites/loader/onLoad/sentryOnLoadError/test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { envelopeRequestParser, waitForErrorRequestOnUrl } from '../../../../utils/helpers';
+
+sentryTest(
+  'sentryOnLoad callback is called before Sentry.onLoad() and handles errors in handler',
+  async ({ getLocalTestUrl, page }) => {
+    const url = await getLocalTestUrl({ testDir: __dirname });
+    const req = await waitForErrorRequestOnUrl(page, url);
+
```

Comment:
l: we could assert on the console output here so that we ensure that the error is logged, wdyt?

Reply:
I updated this to assert on the error message!